src/ui-kit/components/Button/Button.styled.js

```diff
@@ -0,0 +1,38 @@
+import Button from '@mui/material/Button';
+import styled from '@emotion/styled';
+
+export const StyledButton = styled(Button)`
+  padding: 8px 12px;
+  text-transform: none;
+  font-size: ${(p) => (p.fontSize ? p.fontSize : '16px')};
```

Comment:
Maybe it should be  `${p.fontSize}px`

Reply:
No, I just asked what the idea was.

Reply:
I think we pass value with measurement, but it maybe better to always use px